tests/utils.py

```diff
@@ -73,3 +74,14 @@ def stub_open(*args, **kwargs):
 
     with patch("__builtin__.open", stub_open):
         yield mock_open, mock_file
+
+
+def is_base64(test_string: str) -> bool:
+    # Taken from https://stackoverflow.com/a/45928164/3200002
+    try:
+        return (
+            base64.b64encode(base64.b64decode(test_string)).decode("utf-8")
```

Comment:
Nit: `"utf-8"` is redundant here in Py3 and can be safely removed

Reply:
Removed the switched `.decode("utf-8")` -> `.decode()` in a rebase. Thanks!